src/main/java/com/monumental/controllers/MonumentController.java

```diff
@@ -51,17 +61,33 @@ public Monument createMonument(@RequestBody CreateMonumentRequest monumentReques
     }
 
     /**
-     * Get a Monument with the specified ID, if it exists
+     * Get a Monument with the specified ID, if it exists and is active or inactive depending on onlyActive
      * @param id - ID of the Monument to get
      * @param cascade - If true, loads all of the lazy-loaded collections associated with the Monument
+     * @param onlyActive - If true, a 404 will be returned if the specified Monument is inactive. If false, the monument
+     *                 will be returned regardless of whether it's active or inactive.
+     *                 If this is false then the user must be a partner or researcher to view
```

Comment:
Can you correct this comment to be `If this is false then the user must be a partner or above to view`?